docs/library/machine.RTC.rst

```diff
@@ -42,12 +42,18 @@ Methods
 
    Initialise the RTC. Datetime is a tuple of the form:
 
-      ``(year, month, day[, hour[, minute[, second[, microsecond[, tzinfo]]]]])``
+      ``(year, month, day, hour, minute, second, microsecond, tzinfo)``
+
```

Comment:
Probably worth mentioning that `tzinfo` is currently ignored.  Also `microsecond` is ignored by all ports but might be used in the future.

Reply:
Done & updated.